x-pack/plugins/lens/public/app_plugin/app.test.tsx

```diff
@@ -186,8 +186,8 @@ describe('Lens App', () => {
         query: { query: '', language: 'lucene' },
         filters: [pinnedFilter],
         resolvedDateRange: {
-          fromDate: '2021-01-10T04:00:00.000Z',
-          toDate: '2021-01-10T08:00:00.000Z',
+          fromDate: 'now-7d',
```

Comment:
why this change?

Reply:
the resolvedDateRange is calculated using getResolvedDateRange which was updated to not convert the times to absolute https://github.com/elastic/kibana/pull/180440/files/c35163cad9575ce744bc8d0653b60c00fd9f4a08#diff-8d291c5745a38169cdc116550cb26825208d9bc84d2cd6d81cda9254cd15135cR48